server/controllers/stock/core.js

```diff
@@ -441,8 +445,10 @@ async function getDailyStockConsumption(params) {
   filters.dateTo('dateTo', 'date');
   filters.equals('depot_uuid', 'uuid', 'd');
   filters.equals('inventory_uuid', 'uuid', 'i');
-  filters.custom('consumption', '(m.flux_id IN (9, 10) OR (m.flux_id = 8 AND d.is_warehouse = 1))');
+  filters.custom('consumption', `((m.flux_id IN (9,10) OR (m.flux_id=8 AND d.is_warehouse=1))) AND i.consumable=1`);
```

Comment:
Is it possible to have `constants` instead of integer value so that we can quickly figure out what the code have to do. for example : 
```js
const value = `
(
  (m.flux_id IN (${flux.TO_PATIENT}, ${flux.TO_SERVICE}) 
  OR 
  (m.flux_id=${flux.TO_OTHER_DEPOT} AND d.is_warehouse=1)
) AND i.consumable=1
`
filters.custom('consumption', value);
```